src/components/ui/navigation-menu.tsx

```diff
@@ -0,0 +1,161 @@
+import * as React from 'react';
```

Comment:
Consider splitting on separate files. This comment relates to all component files longer than 100 lines.